strapi/design-system
# chore(eslint): add eslint-mdx package and fix eslint errors

### What does it do?
Adds `eslint-mdx` package to the design system and `.mdx` file to the `lint` command.

### Why is it needed?
Without linting mdx files it's very common to have many unused variables imported or just other small bugs. Let's keep it clean ✨ 

### How to test it?

1. Run `yarn lint` & verify if there are any errors or warnings.
2. Make a mistake inside any `mdx` file, run `yarn lint` and verify if an error is shown.
